source/draw/Color.ooc

```diff
@@ -137,6 +142,31 @@ ColorYuv: cover {
 	operator != (other: ColorBgra) -> Bool { !this equals(other) }
 }
 
+ColorYuva: cover {
+	yuv: ColorYuv
+	alpha: UInt8
+	init: func@ (=yuv, =alpha)
+	init: func@ ~uint8 (y, u, v, a: UInt8) {
+		this yuv = ColorYuv new(y, u, v)
+		this alpha = a
+	}
+	init: func@ ~default { this init(0, 128, 128, 255) }
```

Comment:
For covers yes. Keep this one. I don't know how to handle it, especially not in this case.